Makefile

```diff
@@ -11,7 +11,7 @@ BUILDER_ARGS=
 build:
 	echo "Derived image tag: $(DESCRIBE)"
 	echo "Actual image tag: $(IMAGE_TAG)"
-	$(BUILDER) build -f docker/Dockerfile $(BUILDER_ARGS) -t $(IMAGE) .
+	$(BUILDER) build $(BUILDER_ARGS) -t $(IMAGE) .
```

Comment:
Instead of creating a symbolic link for the Docker file, you could specify more build information in the Docker file's service description:
```yaml
build:
  context: .
  dockerfile: docker/Dockerfile
```
Such a change would consider the actual directory structure. Then you could revert the change in this Makefile.